setup.cfg

```
@@ -21,7 +21,7 @@ install_requires =
     astropy>=4.2
     traitlets>=5.0.5
     bqplot>=0.12.26
-    bqplot-image-gl>=1.3.0
+    bqplot-image-gl>=1.3.3
```

My PR requires 1.4.1, so to avoid a future merge conflict...
```suggestion
    bqplot-image-gl>=1.4.1

---

Thanks!